.github/workflows/build_docker.yml

```diff
@@ -20,6 +20,52 @@ on:
     types: [released]
 
 jobs:
+  focal:
```

Comment:
Would love if we could be less repetitive but i guess that's a no.

Reply:
I don't think YAML supports the kind of templating we'd need to fully DRY-ify this file without an intermediate preprocessing step (ie, just to replace the words focal, with xenial). 

However, YAML _does_ support anchors which means in theory we could pull out the common bits, but sadly GH doesnt seem to support these yet. https://github.community/t/support-for-yaml-anchors/16128/38

Reply:
I have found a solution using GitHub Actions own expressions and context syntax. We'll have to revert to opening up a nle-bionic repo if we go for it though.